### PR TITLE
Exported a widgets repository to paint UI

### PR DESCRIPTION
**Pull request by FranciscoGaraventa**
-Change name of class item_model to movie_model
-Change name of class result_model to movie_result

Flutter Analyze result
image

### CODE REVIEW

File: lib/src/screen/main_screen.dart [hunk 1]
```diff
@@ -1,14 +1,14 @@
 import 'package:flutter/material.dart';
 import 'package:flutter/painting.dart';
+import 'package:movies_app/src/styles/routes.dart';
```
Comment (MarcosCGlobant) [inline lib/src/screen/main_screen.dart]: lets use relative import here

File: lib/src/screen/movies_by_genre.dart [hunk 2]
```diff
@@ -25,6 +25,7 @@ class _MoviesGenreList extends State<MoviesByGenre> {
         .fetchMoviesByQuery(MovieEvent.loadMoviesByGenreId, widget.genre.id.toString());
   }
 
+
```
Comment (MarcosCGlobant) [inline lib/src/screen/movies_by_genre.dart]: Remove extra line

File: lib/src/widgets/movies_events.dart [hunk 3]
```diff
@@ -1,7 +1,6 @@
 import 'package:flutter/material.dart';
-import 'empty_state.dart';
-import 'error_state.dart';
-import 'movie_grid.dart';
+import 'package:movies_app/src/styles/routes.dart';
```
Comment (MarcosCGlobant) [inline lib/src/widgets/movies_events.dart]: lets use relative import

File: lib/src/screen/main_screen.dart (hunk 1 above)
Comment (FranciscoGaraventa) [reply lib/src/screen/main_screen.dart]: Done

File: lib/src/screen/movies_by_genre.dart (hunk 2 above)
Comment (FranciscoGaraventa) [reply lib/src/screen/movies_by_genre.dart]: Done

File: lib/src/widgets/movies_events.dart (hunk 3 above)
Comment (FranciscoGaraventa) [reply lib/src/widgets/movies_events.dart]: Done

**Review by MarcosCGlobant — APPROVED** [review]
LGTM

**Pull request merged**